### ISSUE TITLE
Course Attachments API Endpoint - Attachment URL

### ISSUE THREAD

**Issue by seanthepottingshed** [JackHaK/TMF#26]
@JackHaK @PeteHaK @joshgosselin 

The download URL provided by the Course Attachments API endpoint is currently a relative URL and as such these attachments don't exist on the `release` server. 

Going forwards where are these files to be hosted so that they can be downloaded?

**Comment by JazHaK** [comment]
http://gtadev.humbleandkind.co.uk/storage/

**Comment by joshgosselin** [comment]
@seanthepottingshed we haven't 100% confirmed this but last discussions with @PeteHaK was that the production version of the integration tier will live on our digital ocean hosting environment.

**Comment by JackHaK** [comment]
http://gtadev.humbleandkind.co.uk/storage/ is the path

**Comment by seanthepottingshed** [comment]
@JackHaK 

Thanks for the above I will use this for the time being.

**Comment by JazHaK** [comment]
Yes it will - but at the moment the dev version uses this location - can you make it configurable for the time being

**Comment by seanthepottingshed** [comment]
@JackHaK 

Yep, already defining a `baseUrl` dependent on `env('APP_ENV')` ;)

**Comment by JackHaK** [comment]
If you would prefer to get these through an API then we can do that

**Comment by seanthepottingshed** [comment]
@JackHaK 

TBH I'm not sure of the benefit of that at this moment in time, maybe once some of the more fundamental integrations have been taken care of we could revisit.
